<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/game/jackpots/jackpots/woodBG2.jpg);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="0">
    <widget class="QPushButton" name="backB">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(66, 30, 4);</string>
     </property>
     <property name="text">
      <string>BACK</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/buttons/icons/icons/back.png</normaloff>:/buttons/icons/icons/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 0);</string>
     </property>
     <property name="text">
      <string>NEW ACCOUNT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 0);</string>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="password">
     <property name="styleSheet">
      <string notr="true">color: rgb(66, 30, 4);</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="email">
     <property name="styleSheet">
      <string notr="true">color: rgb(66, 30, 4);</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 0);</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="response">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 0);</string>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="passwordConfirm">
     <property name="styleSheet">
      <string notr="true">color: rgb(66, 30, 4);</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 0);</string>
     </property>
     <property name="text">
      <string>Profile Name</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="profileName">
     <property name="styleSheet">
      <string notr="true">color: rgb(66, 30, 4);</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 0);</string>
     </property>
     <property name="text">
      <string>Identity Question</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QComboBox" name="identityQuestionBox">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 0);</string>
     </property>
     <item>
      <property name="text">
       <string>What was the last name of your third grade teacher?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>What was the name of the boy/girl you had your second kiss with?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Where were you when you had your first kiss?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>When you were young, what did you want to be when you grew up?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Who was your childhood hero?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>What is your oldest cousin's first/last name?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>What was the name of your elementary/primary school?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>What are the last 5 digits of your driver's license number?</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 0);</string>
     </property>
     <property name="text">
      <string>Answer</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="answer">
     <property name="styleSheet">
      <string notr="true">color: rgb(66, 30, 4);</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QPushButton" name="registerUser">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(66, 30, 4);</string>
     </property>
     <property name="text">
      <string>REGISTER</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/buttons/icons/icons/confirm.png</normaloff>:/buttons/icons/icons/confirm.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Ignored</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
